src/doc.js

```diff
@@ -88,6 +88,26 @@ var prototype = node.derive({
         this.words = per(characters(runs)).per(split(self.codes)).map(function(w) {
             return word( self.defaultFormatting, w, self.codes);
         }).all();
+
+
+        /**
+         * Carota adds a new word with a new line to mark end-of-document and it's height is not specified
+         * the height should equal to the previous line height
+         */
+        if( this.words && this.words[ this.words.length - 1 ] && this.words[ this.words.length - 2 ] ) {
+            if ( this.words[ this.words.length - 1 ].text.plainText === '\n' ) {
```

Comment:
We can right the condition in single line. No need of two separate if conditions.
`if( this.words && this.words[ this.words.length - 1 ] && this.words[ this.words.length - 2 ] && this.words[ this.words.length - 1 ].text.plainText === '\n' ) {`